src/service/anonymousProfile.ts

```diff
@@ -1,9 +1,8 @@
-import {AnonUser, AnonUserResponse} from '../../types/service/AnonProfile.type';
 import anonymousApi from './anonymousConfig';
+import {AnonUser, AnonUserResponse} from '../../types/service/AnonProfile.type';
 
 export const generateAnonProfileOtherProfile = async (userId): Promise<AnonUser> => {
   try {
-    console.log({userId});
     const result = await anonymousApi.post<AnonUserResponse>(`/chat/users/${userId}`);
     console.log({result: result.data});
     return Promise.resolve<AnonUser>(result?.data.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `console.log` statement on line 6 could potentially expose sensitive user data in an uncontrolled environment. It's generally not a good practice to log sensitive information, especially in production code. If this logging is necessary for debugging purposes, consider using a conditional statement to ensure it only happens in a development environment.

```diff
- console.log({result: result.data});
+ if (process.env.NODE_ENV === 'development') {
+   console.log({result: result.data});
+ }
```
Also, the Promise.resolve() method is redundant here since async functions implicitly wrap the returned value in a Promise. You can simplify the return statement as follows:

```diff
- return Promise.resolve<AnonUser>(result?.data.data);
+ return result?.data.data;
```
Please make sure that the calling function handles the case when `result?.data.data` is undefined or null.


<!-- This is an auto-generated comment by OSS CodeRabbit -->